core/metacling/src/TClingCallbacks.h

```diff
@@ -49,7 +49,7 @@ class TClingCallbacks : public cling::InterpreterCallbacks {
    bool fPPChanged;
    // This vector holds a clang cxxmodules where corresponding library should be loaded in Transaction
    // afterwards.
-   std::vector<clang::Module*> fPendingCxxModules;
+   std::vector<const clang::Decl*> fPendingClangDecls;
```

Comment:
The member name could be better, answering questions like: pending what? Which `Decl`s does this contain? How do they relate to the callback?